src/main/resources/sql/V10__Index_on_index_table.sql

```diff
@@ -0,0 +1,7 @@
+create index if not exists index_name on index (name);
+create index if not exists index_key on index (key);
+create index if not exists index_sha256hex on index (sha256hex);
+create index if not exists index_start_entry_number on index (start_entry_number);
+create index if not exists index_end_entry_number on index (end_entry_number);
+create index if not exists index_start_index_entry_number on index (start_index_entry_number);
+create index if not exists index_end_index_entry_number on index (end_index_entry_number);
```

Comment:
When I added the indexes on all these, we were doing lookups on each of these columns from what I saw over all of our different queries. Happy to revisit this again.